tests/bdd/bdd_test.go

```diff
@@ -36,7 +36,7 @@ import (
 	chaosClient "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned/typed/litmuschaos/v1alpha1"
 )
 
-var kubeconfig = "/home/circleci/.kube/config"
+var kubeconfig = "/home/gitlab-runner/.kube/config"
```

Comment:
Can you use `$HOME` variable instead of defining the home path manually, So that it will work for gitlab as well as circleci.
Like:
```
var kubeconfig = "$HOME/.kube/config"
```

Reply:
lets pass some external arg to the bdd to mention path from where to pick kubeconfig